src/transformers/models/rt_detr/__init__.py

```diff
@@ -61,15 +62,15 @@
     except OptionalDependencyNotAvailable:
         pass
     else:
-        from .modeling_resnet_rt_detr import (
-            RTDetrResNetBackbone,
-            RTDetrResNetPreTrainedModel,
-        )
         from .modeling_rt_detr import (
             RTDetrForObjectDetection,
             RTDetrModel,
             RTDetrPreTrainedModel,
         )
+        from .modeling_rt_detr_resnet import (
```

Comment:
The RT-DETR ResNet backbone can be in a separate folder (I did the same for ViTPose)

Reply:
https://github.com/huggingface/transformers/commit/db2bef18b78a7736ad733bd4538bc7b9ff9c99c6